blocks/default-output-block/package-lock.json

```diff
@@ -13,11 +13,11 @@
       }
     },
     "@babel/generator": {
-      "version": "7.11.0",
-      "resolved": "https://registry.npmjs.org/@babel/generator/-/generator-7.11.0.tgz",
-      "integrity": "sha512-fEm3Uzw7Mc9Xi//qU20cBKatTfs2aOtKqmvy/Vm7RkJEGFQ4xc9myCfbXxqK//ZS8MR/ciOHw6meGASJuKmDfQ==",
+      "version": "7.11.6",
```

Comment:
You should not need to update package dependencies for this ticket. Can you revert the changes on the package-lock. In actuality (for better and for worse), package-lock files aren't published to blocks